src/assets/Enums.ts

```diff
@@ -56,3 +56,9 @@ export const TEST_STATUS = {
     SUBMITTED: "submitted",
     CANCELLED: "cancelled"
 };
+
+export const HGV_TRL_ROADWORTHINESS_TEST_TYPES = {
+    id_hgv_trl : "122",
```

Comment:
conventionally all caps for constants

Reply:
I would have used an array(as I think there are more roadworthiness test-types)